### ISSUE TITLE
Update navigation bar rightButtonImage dynamically 

### ISSUE THREAD

**Issue by octopusbaba** [aksonov/react-native-router-flux#3448]
I have my scene defined inside **Router** as follows:

    <Scene 
             key="latest"
             title="LATEST"
             titleStyle={{flex:0}}
             component={Latest}
             onRight={()=>{}} 
             rightButtonImage={NOTIFICATION_ICON}
             onLeft={()=>{}}
             leftButtonImage={NAV_SEARCH_ICON}
    />

With the above code , I can make navigation action from my Latest screen  .
Which is achieved by using the below code.

```
  componentDidMount() {
          this.props.navigation.setParams({
               'onRight': this.showNotifications,
               'onLeft': this.showSearch,
         })
  }
```

So, to update the rightButtonImage from Latest screen , I tried to add rightButtonImage inside the setParams() method as:

```
 this.props.navigation.setParams({
      'onRight': this.showNotifications,
      'onLeft': this.showSearch,
      'rightButtonImage': {NOTIFICATION_ICON_ON}
    })

```

Basically I want to change my notification bell icon , whenever the new notification arrives . But , by adding 'rightButtonImage' in setParams() doesn't work.

Can anybody please help?
**How can we update our rightButtonImage ?**

**Comment by octopusbaba** [comment]
Actions.refresh({
      right  : this.renderRightButton,
    })


 renderRightButton() {
    return (
      <TouchableOpacity onPress={ console.log(this) } >
        <Image source = {NOTIFICATION_ICON_ON} style = {{marginRight:12}}/>
      </TouchableOpacity>
    )
  }


Thank me later.
